README.md

```diff
@@ -28,30 +90,111 @@ cd comms-gatekeeper
 yarn install
 ```
 
+3. Build the project:
+
+```bash
+yarn build
+```
+
 ### Configuration
 
 The service uses environment variables for configuration. Create a `.env` file in the root directory following the ones defined in the `.env.default` file.
 
-### Starting the Server
+Key configuration variables include:
+
+- `PG_COMPONENT_PSQL_CONNECTION_STRING`: PostgreSQL connection string
+- `LIVEKIT_HOST`, `LIVEKIT_API_KEY`, `LIVEKIT_API_SECRET`: LiveKit configuration
+- `CATALYST_URL`: Catalyst content server URL
+- `PLACES_API_URL`: Places API endpoint
+- `SOCIAL_SERVICE_URL`: Social service endpoint
+- `AWS_SNS_ARN`: SNS topic ARN for event notifications
 
-1. **Start the PostgreSQL database:**
+### Running the Service
+
+#### Setting up the environment
+
+In order to successfully run this server, external dependencies such as databases must be provided.
+
+To do so, this repository provides you with a `docker-compose.yml` file for that purpose. In order to get the environment set up, run:
 
 ```bash
-docker-compose up
+docker-compose up -d
 ```
 
-2. **Start the application:**
+This will start:
+- PostgreSQL database on port `5450`
+
+#### Running in development mode
+
+To run the service in development mode:
 
 ```bash
 yarn start:dev
 ```
 
+This will:
+- Build the TypeScript code
+- Start the server with hot-reload capabilities
+
+For watch mode with automatic rebuilds:
+
+```bash
+yarn dev
+```
+
 ### Database Migrations
 
-The service uses `node-pg-migrate` for database migrations. Check the [documentation](https://salsita.github.io/node-pg-migrate/) for more information.
+The service uses `node-pg-migrate` for database migrations. Migrations are located in `src/migrations/`.
+
+To run migrations:
+
+```bash
+yarn migrate up
+```
+
+To rollback migrations:
+
+```bash
+yarn migrate down
+```
+
+For more information, check the [node-pg-migrate documentation](https://salsita.github.io/node-pg-migrate/).
+
+## Testing
+
+This service includes comprehensive test coverage with both unit and integration tests.
+
+### Running Tests
+
+Run all tests with coverage:
+
+```bash
+yarn test
+```
+
+Run tests in watch mode:
+
+```bash
+yarn test --watch
+```
+
```

Comment:
In the README from the template we've got two more commands, one to run the tests as unitary tests and the other to run them as integration tests:

```
Run only unit tests:

```bash
yarn test test/unit
```

Run only integration tests:

```bash
yarn test test/integration
```

Would you mind adding them to the README?